app/experimenter/nimbus-ui/src/lib/visualization/types.ts

```diff
@@ -19,6 +19,16 @@ export type AnalysisDataWeekly = Exclude<AnalysisData["weekly"], null>;
 export interface Metadata {
   metrics: { [metric: string]: MetadataPoint };
   outcomes: { [outcome: string]: MetadataPoint };
+  external_config?: MetadataExternalConfig | null;
+}
+
+export interface MetadataExternalConfig {
+  end_date: string | null;
+  enrollment_period: string | null;
+  reference_branch: string | null;
+  skip: boolean;
+  start_date: string | null;
+  url: string;
 }
```

Comment:
Maybe just a note to myself, but: I was a little confused looking at [this change to the Jetstream schema](https://github.com/mozilla/jetstream/pull/841) and [this comment](https://github.com/mozilla/experimenter/issues/6129#issuecomment-893778432) in issue #6129.

But, things have been updated since then. [Here's the Jetstream change that corresponds to this type](https://github.com/mozilla/jetstream/commit/530469c5facf9f1573a3882d831e1dd73c6cb3eb).

And here's what the data in a prod experiment looks like from the analysis endpoint

![Capture](https://user-images.githubusercontent.com/21687/130279063-09af836d-1c01-41ef-82ea-6a9cb6140170.PNG)

Reply:
Ack, yeah, I can see why that's confusing. We originally thought this was a problem with only the `reference_branch` and it was exposed in the metadata as only `reference_branch` (jetstream config schema version 3). Then, a few days later, #6200 was filed, which prompted changing this to that `external_config` object (schema version 4).

The metadata for that experiment was re-uploaded manually end of day yesterday by Anna, as well as a couple of others that had overrides, and this morning I went to double check that `external_config` was coming back on `metadata` on the experiment in that bug and we discovered we needed to "force jetstream data fetch" for that experiment since old completed experiments apparently keep that cache around for 3 days or so (I'm making note of this in a doc update).